docs/eval-logs.qmd

```diff
@@ -394,6 +395,58 @@ AWS_DEFAULT_REGION=eu-west-2
 
 One thing to keep in mind if you are storing logs on S3 is that they will no longer be easily viewable using a local text editor. You will likely want to configure a [FUSE filesystem](https://github.com/s3fs-fuse/s3fs-fuse) so you can easily browse the S3 logs locally.
 
+### Azure Blob Storage {#sec-azure}
+
+You can store evaluation logs in Azure Blob Storage using any Azure-compatible fsspec scheme (`az://`, `abfs://`, or `abfss://`). Inspect relies on `fsspec` + `adlfs`, so no code changes are needed beyond installing the Azure dependency.
+
+```
+pip install inspect_ai[dev]  # installs adlfs
```

Comment:
Here we should do a more targeted installation of just the dependencies we need as [dev] will install way too much.